sdks/python/apache_beam/options/value_provider.py

```diff
@@ -69,17 +69,24 @@ def __init__(self, option_name, value_type, default_value):
   def is_accessible(self):
     return RuntimeValueProvider.runtime_options is not None
 
+  @classmethod
+  def get_value(cls, option_name, value_type=None, default_value=None):
```

Comment:
Can we make the `value_type` and `default_value` args required. I do not think they need to be optional.

Reply:
Done.